src/components/sidebar/sidebar.component.js

```diff
@@ -17,33 +17,22 @@ class Sidebar extends Modal {
     );
   }
 
-  onButtonKeyDown = (ev) => {
-    if (Events.isEnterKey(ev) || Events.isSpaceKey(ev)) {
-      ev.preventDefault();
-      this.props.onCancel();
-    }
-
-    return null;
-  }
-
-  /** Returns the markup for the close icon. */
-  get closeButton() {
-    if (this.props.onCancel) {
-      return (
-        <SidebarCloseStyle>
-          <Icon
-            className='carbon-sidebar__close-icon'
-            data-element='close'
-            onClick={ this.props.onCancel }
-            type='close'
-            tabIndex='0'
-            role='button'
-            onKeyDown={ this.onButtonKeyDown }
-          />
-        </SidebarCloseStyle>
-      );
-    }
-    return null;
+  closeIcon = () => {
```

Comment:
I suggest to replace this array method with normal class method just to be consistent in this class

Reply:
fixed: fb7671f